### DISCUSSION TITLE
How to make the Starship prompt run in only the pty console and not on tty console?

### DISCUSSION THREAD

**Discussion by diyfan** [starship/starship#3926] [Q&A] [answered]
Shell: Fish
OS   : Arch GNU/Linux

I didn't like the prompt on the VGA console (which is what we get with CTRL+ALT+F3 or F5) so I want the normal fish prompt in the VGA console.

**Comment by davidkna** [comment, marked as answer]
Something like this could work (may need to be adjusted for your specific `$TERM` variable):
```sh
if test "$TERM" != "dumb" 
    starship init fish | source
end
```

**Reply by diyfan** [reply]
thanks ; but it did not work

**Reply by MuhammedZakir** [reply]
I do this in my ZSH config. It should work as long you don't source `starship` init function from other places.

Just to double-check: did you use the correct `$TERM` value? In my case, it is `linux`.

**Reply by diyfan** [reply]
Yeah it did work for me now ; seems like I should have done 'linux' instead of 'alacritty'. Thanks for the help.
